src/operating-hours.ts

```diff
@@ -13,6 +11,7 @@ export const getOperatingHours = async (): Promise<OperatingHoursState> => {
     },
   };
 
+  const {SERVERLESS_URL} = require('../private/secret')
```

Comment:
Can you leave a small comment on this one just so remember why it's done this way? Also, if you can please add the ending `;` just to b consistent :P